### DISCUSSION TITLE
Internal Client Timeout

### DISCUSSION THREAD

**Discussion by Rich968** [hsaturn/TinyMqtt#40] [Q&A]
I have noticed that the internal client when connected to the internal broker times out after 60 seconds. How can I keep the internal client connected indefinitely? I have used your client-with-wifi example modified to use just one client.

**Comment by hsaturn** [comment]
Hello Rich, this problem should be solved with the latest release (0.9.18).
Feel free to open an issue, that's the right place for such question (and I try to answer very quickly there).

Best regards
